docs/opencompetition.md

```diff
@@ -0,0 +1,116 @@
+# Open Optimization Competition 2020
```

Comment:
I would tend to call this file `opencompetition2020.md` instead, to be forward compatible. Then again, in 2021 we would probably delete it so it does not change much

Reply:
Done.